### DISCUSSION TITLE
[blocks]: Time Picker

### DISCUSSION THREAD

**Discussion by osmanmaniops18** [shadcn-ui/ui#4226] [Blocks Request]
### Description

The Time Picker component allows users to select a time from a dropdown menu. It will be used in a web-based scheduling application.

**Comment by geudrik** [comment]
https://time.openstatus.dev/

I composed mine to look like this

![image](https://github.com/shadcn-ui/ui/assets/2020115/8a5fbde7-f3a5-4b5b-91ef-9c3bff367094)

**Comment by osmanmaniops18** [comment]
How you do that

**Comment by Maliksidk19** [comment]
https://shadcn-datetime-picker.vercel.app/

try this

**Reply by osmanmaniops18** [reply]
thanks
